cmd/juju/commands/switch.go

```diff
@@ -74,9 +80,28 @@ func (c *switchCommand) Info() *cmd.Info {
 	})
 }
 
+// SetFlags implements Command.SetFlags.
+func (c *switchCommand) SetFlags(f *gnuflag.FlagSet) {
+	c.CommandBase.SetFlags(f)
+	// The `-:` requires a special handling because it will be detected as a flag by command library
+	f.BoolVar(&c.IsPreviousController, ":", false, "Switch to previous controller")
```

Comment:
:vomiting_face:

Reply:
This results in:
```
Command Options:
-:  (= false)
    Switch to previous controller
```
in help text. Is there a way this can be suppressed?

Reply:
Yes... I know. And agree :D

I can try to dig into the cmd libraries (I know how to do this kind of magic in `spf13/cobra` but in the lib we use... I will dig into it.

Reply:
There is no way to prevent it, unfortunately. @jack-w-shaw 

To avoid this ugliness we can:

- use another character than `-` as placeholder for previous (but it would make it less intuitive)
- update https://github.com/juju/gnuflag to have "hidden flag" (flag that doesn't show up in automatec documentation)
- update https://github.com/juju/cmd to be able to catch up argument before flag parsing and be able to do some smuggle with args before actually parse argument (or be able to deactivate arg parsing)